<launch>
    <node name="yolo11_node" pkg="yolo11_ros" type="yolo11_ros_node" output="screen">
        <param name="model_path" value="/home/orangepi/NECDFlight2025/src/yolo11_ros/model/point_s_fix_all.rknn"/>
        <param name="camera_index" value="0"/>
    </node>
</launch>